modules/ROOT/pages/snippets/add_channels_scc.adoc

```diff
@@ -11,5 +11,6 @@ spacewalk-common-channels \
 . Synchronize the channels:
 +
 ----
-mgr-sync refresh --refresh-channels
+spacewalk-repo-sync
```

Comment:
If this is `add_channels_scc`, why do you use `spacewalk-repo-sync` here instead of `mgr-sync`?

Reply:
Because for OSs that use spacewalk-channel-sync to add channels, they use spacewalk-repo-sync to synchronize:

![image](https://user-images.githubusercontent.com/3914967/98317755-d177c480-2028-11eb-859a-d3c0b7e972ec.png)

Reply:
> Because for OSs that use spacewalk-channel-sync to add channels, they use spacewalk-repo-sync to synchronize:

This is true, but the filename if this `adoc` is `add_channels_scc`, which doesn't make sense if you are explaining how to use `spacewalk-repo-sync` for OS that use `spacewal-common-channel`, right?

How is this related to "adding channels from SCC"?

Reply:
It's not related. We have four different snippets for add channels:

* `add_channels_mgrsync.adoc`
* `add_channels_scc_multiarch.adoc`
* `add_channels_scc.adoc`
* `add_channels_wizard.adoc`

So, if you want to use the SUSE Customer Center on the WebUI, it's `add_channels_wizard.adoc`, and if you want to use that method on the CLI, it's `add_channels_mgrsync.adoc`. I don't want to use `spacewalk-common-channels` in the filenames of the snippets because it's too long. Since we don't call the SUSE Customer Center methods "SCC" in any context, it makes sense to use that abbreviation for `spacewalk-common-channels`. None of this is customer facing, it's just used internally.

Reply:
*Now* I know it's not related. But using the acronym `scc` for `spacewalk-common-channels` is just not right.

I got confused, and everyone else will get confused, including me when I see it the next time and I don't remember about this discussion :-)

Reply:
Lol, OK! How do you feel about `swcc`?